isis/src/control/objs/BundleUtilities/BundleUtilities.truth

```diff
@@ -501,9 +501,7 @@ TWI  (dd)          0.00000000            0.00000000               0.00000000
 index =  "1"
 instrument id =  "InstrumentId2"
 number parameters =      "3"
-number position param =  "0"
-number pointing param =  "3"
-parameter list:  ()
+parameter list:  ("X", "Y", "Z", "RA", "DEC", "TWIST")
```

Comment:
There's a weird thing with the ISIS observations where you always have x,y,z,ra,dec,twist because we always output the center value even if we don't solve for them.

Reply:
Thanks for the explanation! That really confused me.